### PR TITLE
PE 32314 upgrade ember

### PR DESCRIPTION
**Pull request by KNBAnderson**
This commit will upgrade ember to 3.24.3; the current LTS version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,14 +37,21 @@
...
+    "commitizen": {
+      "path": "node_modules/cz-conventional-changelog"
+    }
+  },
   "dependencies": {
-    "ember-cli-babel": "^7.21.0",
-    "ember-cli-htmlbars": "^5.2.0"
+    "ember-cli-babel": "^7.26.6",
+    "ember-cli-htmlbars": "^5.2.0",
+    "popper.js": "^1.16.1"
```
Comment (jonathannewman) [inline package.json:48]: what pulled in popper.js?

Comment (KNBAnderson) [reply package.json:48]: I'm not sure, one of the codemods

Comment (KNBAnderson) [reply package.json:48]: Als, what is that CLA comment above? Do I need to sign up for it?

Comment (jonathannewman) [reply package.json:48]: Yes, you need to sign up for it.  I don't think we need popper.  That isn't shipped with Ember.  Not sure why it would be there.

**Comment by jonathannewman** [comment]
I think we need to regenerate the yarn.lock file too.  It still seems to have popper.js in it.

**Comment by KNBAnderson** [comment]
I didnt see popper in the yarn lock locally, but I deleted the node modules and reran yarn. Are you still seeing it?

**Pull request merged**